catchpoint-mixin/targets.libsonnet

```diff
@@ -9,4 +9,111 @@ local utils = commonlib.utils {
   new(this): {
     local vars = this.grafana.variables,
     local config = this.config,
+    local testNameLabel = config.instanceLabels + config.testNameLabel,
+
+    topAvgLoadTimeTestName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk by (test_name) (2, avg by (instance, test_name) (catchpoint_load_time{%(testNameSelector)s}))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    topMaxTotalLoadTime:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk by (test_name) (2, max by (instance, test_name) (catchpoint_load_time{%(testNameSelector)s}))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    topAvgDocumentCompletionTimeTestName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk by (test_name) (2, avg by (instance, test_name) (catchpoint_document_complete_time{%(testNameSelector)s}))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    topMaxDocumentCompletionTime:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk by (test_name) (2, max by (instance, test_name) (catchpoint_document_complete_time{%(testNameSelector)s}))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    topAvgRequestSuccessRatioTestName:
```

Comment:
For request ratio, we actually do not want the top but the bottom. Looking at past implementations, it looks like we say `Bottom` instead and take a `bottomk` to find the worst performing request ratio. https://github.com/grafana/jsonnet-libs/blob/60f98f603ffb2f88b360d9e31dafa5b0422914a7/couchbase-mixin/dashboards/couchbase-bucket-overview.libsonnet#L470